app/controllers/institutions_controller.rb

```diff
@@ -128,15 +128,15 @@ def new_from_invite_data
     end
   end
 
+  private
```

Comment:
`accept_pending_invite` is not a public controller method, so moving it into the private section.